interpreter/cling/tools/CMakeLists.txt

```diff
@@ -6,11 +6,14 @@
 # LICENSE.TXT for details.
 #------------------------------------------------------------------------------
 
-
 if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/../lib/UserInterface/textinput
    OR CLING_INCLUDE_TESTS)
   add_subdirectory(driver)
   add_subdirectory(Jupyter)
   add_subdirectory(libcling)
   add_subdirectory(demo)
 endif()
+
+if(CLING_BUILD_PLUGINS)
```

Comment:
Do we really need to make this conditional?

Reply:
The intend was to not force cling standalone users to use this as it is quite likely that they won't need it.